app/src/main/java/com/android/unio/ui/home/Home.kt

```diff
@@ -68,16 +71,14 @@ fun EventListOverviewPreview() {
   HomeScreen(
       NavigationAction(NavHostController(LocalContext.current)),
       eventListViewModel = eventListViewModel,
-      onAddEvent = {},
-      onEventClick = {})
+      eventViewModel = PreviewEventViewModel(MockEventRepository(), MockUserRepository()))
 }
 
 @Composable
 fun HomeScreen(
     navigationAction: NavigationAction,
     eventListViewModel: EventListViewModel = viewModel(factory = EventListViewModel.Factory),
-    onAddEvent: () -> Unit,
-    onEventClick: (Event) -> Unit
```

Comment:
Thanks for this removal. It was confusing when there was the foundation of an "add event" logic, while we had discussed that we would put a button that leads to the map.